.github/workflows/ci.yml

```diff
@@ -80,7 +80,7 @@ jobs:
             CONFIGURATION="--configuration phpunit-dama-doctrine.xml.dist"
           fi
 
-          vendor/bin/simple-phpunit -v ${CONFIGURATION}
+          vendor/bin/simple-phpunit ${CONFIGURATION}
```

Comment:
if you're OK, I'd like to keep it this way, the `-v` flag display all the skipped tests, which makes lot of noise in the CI results